app/src/main/java/com/github/studydistractor/sdp/login/FirebaseLoginAuth.kt

```diff
@@ -11,6 +12,7 @@ import javax.inject.Inject
 class FirebaseLoginAuth @Inject constructor() : LoginAuthInterface {
     lateinit var auth : FirebaseAuth
     override fun loginWithEmail(email: String, password: String): Task<AuthResult> {
+        auth = FirebaseAuth.getInstance();

```

Comment:
I think this change is not necessary.

Reply:
I'm getting errors that auth is null

Reply:
How do you handle the init of auth otherwise @bluedrack ?

Reply:
No, it is the good fix but I tought it was not the right commit to fix it. Nevermind